loader_hub/trello/main.py

```diff
@@ -0,0 +1,49 @@
+"""Trello reader."""
+from typing import List
+
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+from trello import TrelloClient
```

Comment:
would you be able to do a lazy import? it's failing tests rn 

(our unit test structure is a bit hacky)

Reply:
Oh, I understand. I looked at examples of other integrations and now I think I've done it right.